### PR TITLE
Controls header and deleted 'c' and 't' cases

### PR DESCRIPTION
**Pull request by Luis-gd**

### CODE REVIEW

File: src/controls.h [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// movement keys define
+
+
+//Normal movements
+const int MOVE_W = 'j';
```
Comment (JohnathonNow) [inline src/controls.h]: This is my fault for choosing an unclear example, but my intent with `MOVE_W` was that it mean `MOVE_WEST`, so the other directions would be `MOVE_S`, `MOVE_E`, and `MOVE_N`, with diagonals being `MOVE_NW`, `MOVE_SE`, `MOVE_NE`, and `MOVE_SW`.

File: src/controls.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+const int MOVE_D = 'l';
+
+//diagonal movement
+const int MOVE_A_W = 'b';
+const int MOVE_D_W = 'n';
+const int MOVE_A_S = 'y';
+const int MOVE_W_S = 'u';
+
+//actions
+const int KEY_< = '<';
```
Comment (JohnathonNow) [inline src/controls.h]: I don't believe `KEY_>` is a valid identifier in C. Perhaps `KEY_CLIMB_UP` and `KEY_CLIMB_DOWN` would be better.

Comment (Luis-gd) [reply src/controls.h]: OK, I will change it

**Comment by Luis-gd** [comment]
I have changed the key name's

**Pull request merged**